pkg/controller/plan/kubevirt.go

```diff
@@ -235,27 +266,29 @@ func (r *KubeVirt) DeleteDataVolumes(vm *plan.VMStatus) (err error) {
 	return
 }
 
-// Delete the importer pod for a PersistentVolumeClaim.
-func (r *KubeVirt) DeleteImporterPod(pvc core.PersistentVolumeClaim) (err error) {
-	var pod *core.Pod
+// Delete the importer pods for a PersistentVolumeClaim.
+func (r *KubeVirt) DeleteImporterPods(pvc core.PersistentVolumeClaim) (err error) {
+	var pods []core.Pod
 	var found bool
-	pod, found, err = r.GetImporterPod(pvc)
+	pods, found, err = r.GetImporterPods(pvc)
```

Comment:
how about
```suggestion
	pods, found, err := r.GetImporterPods(pvc)
```
and drop the variable declarations above?